src_c/event.c

```diff
@@ -144,6 +144,19 @@ RemovePending_PGS_VIDEORESIZE_Events(void * userdata, SDL_Event *event)
     return 1;
 }
 
+static int SDLCALL
+RemovePending_PGS_VIDEOEXPOSE_Events(void * userdata, SDL_Event *event)
+{
+    SDL_Event *new_event = (SDL_Event *)userdata;
+
+    if (event->type == SDL_VIDEOEXPOSE &&
+        event->window.windowID == new_event->window.windowID) {
+        /* We're about to post a new size event, drop the old one */
```

Comment:
This copy-pasted comment is not correct here.

Reply:
There you go, fixed it